internal/backend/db/dbgorm/users.go

```diff
@@ -89,3 +93,24 @@ func (gdb *gormdb) UpdateUser(ctx context.Context, u sdktypes.User, fm *sdktypes
 			Error,
 	)
 }
+
+func (gdb *gormdb) BatchGetUsers(ctx context.Context, uids []sdktypes.UserID) ([]sdktypes.User, error) {
+	q := gdb.db.WithContext(ctx).Where("user_id in ?", kittehs.Transform(uids, func(uid sdktypes.UserID) uuid.UUID { return uid.UUIDValue() }))
+
+	var rs []scheme.User
+	err := q.Find(&rs).Error
```

Comment:
This is where I would expect to see a hard limit on the number of elements that can be requested

Reply:
done